deps/rabbit/priv/schema/rabbit.schema

```diff
@@ -1733,6 +1733,19 @@ end}.
     end
 }.
 
+
+% ===============================
+% AWS section
+% ===============================
+
+%% Whether or not to prefer IMDSv2 when querying instance metadata service (https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/configuring-instance-metadata-service.html).
+%% If not set or set to true, IMDSv2 will be preferred to use first. If fails, IMDSv1 will be used.
+%% {aws_prefer_imdsv2, false}
+
+{mapping, "aws_prefer_imdsv2", "rabbit.aws_prefer_imdsv2",
```

Comment:
Can we namespace this the same way [other AWS peer discovery keys](https://www.rabbitmq.com/cluster-formation.html#peer-discovery-aws) are namespaced? For example, `cluster_formation.aws.prefer_imdsv2` instead of `aws_prefer_imdsv2`.

Reply:
Hi Micheal, thank you for your quick feedback. 

I was thinking that this new `aws_prefer_imdsv2` flag is used in the generic [`rabbitmq_aws` application](https://github.com/rabbitmq/rabbitmq-server/tree/master/deps/rabbitmq_aws) which the `rabbit_peer_discovery_aws` plugin relies on. It is independent from peer discovery plugins and/or cluster formation. 

For example, in future there might be new plugins need interaction with AWS for other purposes such as publishing metrics/logs directly to CloudWatch, and It could be run from a single instance RabbitMQ fleets and/or clusters may not use the `rabbit_peer_discovery_aws` plugin.  

Please feel free to suggest a better namespace for this new flag. And in case `cluster_formation.aws.prefer_imdsv2` still better, please let me know. 

P/S: I accidentally submitted this PR request while preparing a private one for being reviewed internally first, hence I have closed it. I will submit a new PR request once it is approved internally and will make sure your comments here is addressed.

Reply:
I would not object to `aws.prefer_imdsv2` or `amws.preferred_imds = v2` or similar. I have a slight preference towards the latter but recognize that many tools use settings such as `prefer X` which is a boolean (e.g. the IPv6 vs. IPv4 preference on the JVM).

Reply:
Sure. I will change to `aws.prefer_imdsv2 = true | false` then. Thanks again.